core/src/main/scala/org/apache/spark/api/python/PythonUtils.scala

```diff
@@ -27,13 +27,15 @@ import org.apache.spark.SparkContext
 import org.apache.spark.api.java.{JavaRDD, JavaSparkContext}
 
 private[spark] object PythonUtils {
+  val PY4J_ZIP_NAME = "py4j-0.10.8.1-src.zip"
+
   /** Get the PYTHONPATH for PySpark, either from SPARK_HOME, if it is set, or from our JAR */
   def sparkPythonPath: String = {
     val pythonPath = new ArrayBuffer[String]
     for (sparkHome <- sys.env.get("SPARK_HOME")) {
       pythonPath += Seq(sparkHome, "python", "lib", "pyspark.zip").mkString(File.separator)
       pythonPath +=
-        Seq(sparkHome, "python", "lib", "py4j-0.10.8.1-src.zip").mkString(File.separator)
+        Seq(sparkHome, "python", "lib", PY4J_ZIP_NAME).mkString(File.separator)
```

Comment:
BTW, the way of getting Spark home is different in the places. Looks deduplicating the name only is enough since that's the problematic part frequently changing.